.github/workflows/prerelease.yml

```
@@ -50,7 +50,7 @@ jobs:
         run: pnpm i
 
       - name: Bump snapshot
-        run: pnpm changeset version --snapshot ${{ github.head_ref }} --ignore fdm-docs
+        run: pnpm changeset version --snapshot ${{ github.head_ref }}
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)